parlai/tasks/taskmaster/tm_utils.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from .build import build
+import os
+
+###############################################################
+#                                                             #
+# Constants                                                   #
+#                                                             #
+###############################################################
+FIRST_USER = 0
```

Comment:
it might be clearer if you append `_IDX` to each of these, as they are meant for accessing